sdl_ios/SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -269,6 +279,12 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self name:UIApplicationWillEnterForegroundNotification object:nil];
```

Comment:
All NSNotificationCenter observations can be removed from an observer by calling `[[NSNotificationCenter defaultCenter] removeObserver:self]`, which will clean up a few lines of code and make sure you don't miss removing any observations at dealloc time.